packages/ui/build/plugins/component-v-bind-fix.ts

```diff
@@ -0,0 +1,92 @@
+import { Plugin } from 'vite'
+import kebabCase from 'lodash/kebabCase'
+
+const parseVBinds = (style: string) => {
+  return style
+    .match(/v-bind\((.*)\)/g)
+    ?.map((line) => line.match(/v-bind\(([^)]*)\)/)![1]) ?? []
+}
+
+const getRootNodeCode = (code: string) => {
+  const template = code.match(/<template[^>]*>([\s\S]*)<\/template>/)?.[1]
+  const rootNode = template?.match(/<[^>]*>/)?.[0]
+  return rootNode
+}
+
+const renderCssVariablesString = (vBinds: string[]) => {
+  return vBinds.map((vBind, index) => {
+    return `--va-${index}-${kebabCase(vBind)}: \${String(${vBind})}`
+  }, '').join(';')
+}
+
+const renderObjectGuard = (styleContent: string, binds: string[]) => {
+  const cssVariablesString = renderCssVariablesString(binds)
+  const cssVariablesObject = binds.map((vBind, index) => {
```

Comment:
Result will be string, no?

Reply:
It is as string that contains the rendered Object. Renamed.